packages/eslint-plugin-react-hooks/package.json

```diff
@@ -32,12 +32,20 @@
   },
   "devDependencies": {
     "@babel/eslint-parser": "^7.11.4",
+    "@tsconfig/strictest": "^2.0.5",
     "@typescript-eslint/parser-v2": "npm:@typescript-eslint/parser@^2.26.0",
     "@typescript-eslint/parser-v3": "npm:@typescript-eslint/parser@^3.10.0",
     "@typescript-eslint/parser-v4": "npm:@typescript-eslint/parser@^4.1.0",
     "@typescript-eslint/parser-v5": "npm:@typescript-eslint/parser@^5.62.0",
+    "@types/eslint": "^8.56.12",
+    "@types/estree": "^1.0.6",
+    "@types/estree-jsx": "^1.0.5",
+    "@types/node": "^20.2.5",
     "babel-eslint": "^10.0.3",
     "eslint-v7": "npm:eslint@^7.7.0",
-    "eslint-v9": "npm:eslint@^9.0.0"
+    "eslint-v9": "npm:eslint@^9.0.0",
+    "jest": "^29.5.0",
+		"tsup": "^8.3.5",
```

Comment:
```suggestion
	"tsup": "^8.3.5",
```

Reply:
How weird.  I wonder why my IDE didn't pick up the repo's prettier settings in this on line 😅  I've fixed it.